.github/workflows/scripts/pre-commit/venv.sh

```
@@ -6,17 +6,11 @@
 # Used by pre-commit hooks to run Python-based CI checks locally.
 #
 
-set -eu
+set -euo pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)